assets/presto_clp/Dockerfile

```
@@ -0,0 +1,62 @@
+FROM presto/prestissimo-dependency:ubuntu-22.04-presto-0.293
+
+# Install necessary packages
+RUN apt-get update;
```

Hadolint (2.12.0)

[info] Delete the apt-get lists after installing something

(DL3009)